tap_megaphone/schemas/campaign_order_advertisements.schema.json

```diff
@@ -59,13 +59,7 @@
         "null"
       ]
     },
-    "orderID": {
-      "type": [
-        "string",
-        "null"
-      ]
-    },
-    "promoID": {
+    "orderId": {
```

Comment:
Just to confirm since this wasn't mentioned in the description - promoID is being removed from this schema. That's intentional?

Reply:
yes sorry I should have specified! That field isn’t in the api response (I’m guessing megaphone removed it at some point)